com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/security/token/DummyTokenParser.java

```diff
@@ -12,7 +12,7 @@
 import org.springframework.stereotype.Component;
 
 import com.sap.cloud.lm.sl.cf.client.util.TokenFactory;
-import com.sap.cloud.lm.sl.cf.core.util.Configuration;
```

Comment:
Same. Please merge it into the first commit, so that the project is buildable on each commit. :)